run_scans.py

```diff
@@ -22,7 +22,7 @@ def launchTestsArgs(tool, shelf, slot, link, chamber, vfatmask, scanmin, scanmax
   #Build Commands
   setupCmds = []
   preCmd = None
-  cmd = ["%s"%(tool),"-s%d"%(slot),"-g%d"%(link),"--shelf=%i"%(shelf), "--nevts=%i"%(nevts), "--vfatmask=%s"%(vfatmask)]
+  cmd = ["%s"%(tool),"-s%i"%(slot),"-g%i"%(link),"--shelf=%i"%(shelf), "--nevts=%i"%(nevts), "--vfatmask=%i"%(vfatmask)]
```

Comment:
You've (possibly) misunderstood, once the number is extracted from the `chamberInfo.py` it is an `int`, so it's machine readable, then when you pass this to the line we're discussing, it will be correctly converted to `hex`.
Though I'd double check with some test, just to convince yourself